### PR TITLE
Add air-gapped installation guide on K8s

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md [hunk 1]
```diff
@@ -1,10 +1,409 @@
...
+   curl -L -O https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/images-list.txt
+   ```
+
+   {{< notice note >}}
+
+   This file lists images under `##+modulename` based on different modules. You can add your own images to this file following the same rule. To view the complete file, see [Appendix](../install-ks-on-linux-airgapped/#image-list-of-kubesphere-v300).
+
+   {{</ notice >}} 
+
+2. Download `images-manager.sh`.
```
Comment (pixiake) [inline content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md:102]: This should be changed to `offline-installation-tool.sh`.

File: content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md [hunk 2]
```diff
@@ -1,10 +1,409 @@
...
+     -b                     : save kubernetes' binaries.
+     -d IMAGES-DIR          : the dir of files (tar.gz) which generated by `docker save`. default: /home/ubuntu/kubesphere-images
+     -l IMAGES-LIST         : text file with list of images.
+     -r PRIVATE-REGISTRY    : target private registry:port.
+     -s                     : save model will be applied. Pull the images in the IMAGES-LIST and save images as a tar.gz file.
+     -v KUBERNETES-VERSION  : download kubernetes' binaries. default: v1.17.9
+     -h                     : usage message
+   ```
+
+5. Pull images in the file `images-list.txt`.
```
Comment (pixiake) [inline content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md:132]: This should be changed to offline-installation-tool.sh.

Comment (ruiyaoOps) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md:132]: ok , I will change this.

Comment (ruiyaoOps) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md:132]: OK, I will try this.

File: content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md (hunk 1 above)
Comment (Sherlock113) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md]: Changed.

File: content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md (hunk 2 above)
Comment (Sherlock113) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md]: Changed.

**Pull request merged**